tests/osaka/eip7594_peerdas/spec.py

```diff
@@ -0,0 +1,31 @@
+"""Defines EIP-7594 specification constants and functions."""
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class ReferenceSpec:
+    """Defines the reference spec version and git path."""
+
+    git_path: str
+    version: str
+
+
+ref_spec_7594 = ReferenceSpec("EIPS/eip-7594.md", "45d03a84a8ad0160ed3fb03af52c49bd39e802ba")
+
+
+@dataclass(frozen=True)
+class Spec:
+    """
+    Parameters from the EIP-7594 specifications as defined at
+    https://eips.ethereum.org/EIPS/eip-7594.
+    """
+
+    FIELD_ELEMENTS_PER_BLOB = 4096
+    BYTES_PER_FIELD_ELEMENT = 32
+    BYTES_PER_BLOB = FIELD_ELEMENTS_PER_BLOB * BYTES_PER_FIELD_ELEMENT
+
+    # below are mine, not sure if they are defined in specs somewhere
+    KZG_PROOF_LENGTH = 48
+    KZG_COMMITMENT_LENGTH = 48
+    CELL_LENGTH = 2048
```

Comment:
There are constants for these in the specs! Please see:

* [`BYTES_PER_COMMITMENT`](https://github.com/ethereum/consensus-specs/blob/cc6996c22692d70e41b7a453d925172ee4b719ad/specs/deneb/polynomial-commitments.md?plain=1#L83)
* [`BYTES_PER_PROOF`](https://github.com/ethereum/consensus-specs/blob/cc6996c22692d70e41b7a453d925172ee4b719ad/specs/deneb/polynomial-commitments.md?plain=1#L84)
* [`BYTES_PER_CELL`](https://github.com/ethereum/consensus-specs/blob/cc6996c22692d70e41b7a453d925172ee4b719ad/specs/fulu/polynomial-commitments-sampling.md?plain=1#L101)

These are not exposed to the bindings, but maybe they should be. I can do this later if you want.